docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -311,6 +323,16 @@ tcld login
 tcld apikey delete --id <api-key-id>
 ```
 
+### Rotate a Service Account API key
+
+Temporal API keys automatically expire.
+Their maximum life is 90 days.
+Follow these steps to rotate API keys:
+
+1. Delete the existing key.
```

Comment:
Same comment as above for steps to rotate

Reply:
👍